client/components/repository/Workflow/SelectStatus.vue

```diff
@@ -1,20 +1,26 @@
 <template>
-  <v-select v-on="$listeners" v-bind="$attrs" clearable>
+  <v-select v-on="$listeners" v-bind="{ ...$attrs, itemText, itemValue }">
     <template #selection="{ item }">
       <div class="selection d-flex flex-nowrap">
         <v-icon :color="item.color" small class="mr-2">mdi-circle</v-icon>
-        <span class="text--default text-truncate">{{ item.text }}</span>
+        <span class="text--default text-truncate">{{ item.label }}</span>
```

Comment:
If the `itemValue` is provided and its value is not `label`, will this break?

If that's the case, we could fix it with:
```suggestion
        <span class="text--default text-truncate">{{ item[itemText] }}</span>
```